packages/stacks-docs/product/components/textarea.html

```diff
@@ -11,13 +12,17 @@
 {% highlight html %}
 <div class="d-flex fd-column gy4">
     <label class="flex--item s-label" for="example-item">Question body</label>
-    <textarea class="flex--item s-textarea" id="example-item" placeholder="…"></textarea>
+    <div class="d-flex ps-relative">
+        <textarea class="flex--item s-textarea" id="example-item" placeholder="…"></textarea>
+    </div>
 </div>
 {% endhighlight %}
         <div class="stacks-preview--example">
             <div class="d-flex gy4 fd-column">
                 <label class="flex--item s-label" for="example-item7">Question body</label>
-                <textarea class="flex--item s-textarea" id="example-item7" placeholder="e.g. The Stack Overflow glyph used to have 6 bars, but now…"></textarea>
+                <div class="d-flex ps-relative">
+                    <textarea class="flex--item s-textarea ps-relative" id="example-item7" placeholder="e.g. The Stack Overflow glyph used to have 6 bars, but now…"></textarea>
+                </div>
```

Comment:
We should simplify this example by removing the wrapping `div` here.

```suggestion
                <textarea class="s-textarea" id="example-item7" placeholder="e.g. The Stack Overflow glyph used to have 6 bars, but now…"></textarea>
```